docs/pages/versions/unversioned/sdk/firebase-recaptcha.md

```diff
@@ -421,43 +412,37 @@ const styles = StyleSheet.create({
 />
 ```
 
-If you want a custom look & feel, then create your own `<Modal>` or display the `<FirebaseRecaptcha>` component inline in your screen. Make sure to reserve enough space for the widget as it can not only display the compact "I'm not a robot" UI but also the **full verification UI requiring users to select images**.
+If you want a custom look and feel, then create your own `<Modal>` or display the `<FirebaseRecaptcha>` component inline in your screen. Make sure to reserve enough space for the widget as it can not only display the compact "I'm not a robot" UI but also the **full verification UI requiring users to select images**.
 
-```tsx
+```jsx
 import { FirebaseRecaptcha, FirebaseRecaptchaVerifier } from 'expo-firebase-recaptcha';
 
-class CustomPhoneAuthScreen extends React.Component {
-  state = {
-    recaptchaToken: ''
-  };
+function CustomPhoneAuthScreen () {
+  const [recaptchaToken, setRecaptchaToken] = React.useState('');
 
-  onPressSendVerificationCode = async () => {
+  async function onPressSendVerificationCode () {
```

Comment:
```suggestion
  async function onPressSendVerificationCode() {
```

In this notation there should not be space between function name and parenthesis. Alternative:

```suggestion
  const onPressSendVerificationCode = async () => {
```

Reply:
The space was my mistake. Sorry.